src/missed_spans.rs

```diff
@@ -144,8 +144,14 @@ impl<'a> FmtVisitor<'a> {
                     line_start = offset + subslice.len();
 
                     if let Some('/') = subslice.chars().skip(1).next() {
-                        // Add a newline after line comments
-                        self.buffer.push_str("\n");
+                        // check that there are no contained block comments
+                        if !subslice.chars()
+                            .filter(|&c| c != '\t' && c != ' ')
+                            .collect::<String>()
```

Comment:
Not just leading, but all whitespace besides newlines, as `subslice` in this case would be `// comment \n /* block comment */`, which trim wouldn't work on. Although I guess I could also split by newlines and check the start of every line, which might be a little bit faster?